app/models/concerns/form526_workflow.rb

```diff
@@ -0,0 +1,172 @@
+# frozen_string_literal: true
+
+module Form526Workflow
+  extend ActiveSupport::Concern
+
+  FORM_526 = 'form526'
+  FORM_526_UPLOADS = 'form526_uploads'
+  FORM_4142 = 'form4142'
+  FORM_0781 = 'form0781'
+  FORM_8940 = 'form8940'
+  FLASHES = 'flashes'
+  SUBMIT_FORM_526_JOB_CLASSES = %w[SubmitForm526AllClaim SubmitForm526].freeze
+
+  class EvssSubmission
```

Comment:
The 3 classes in this file could be extracted into separate files in `app/services` directory, if desired.

Reply:
To rephrase [my other comment](https://github.com/department-of-veterans-affairs/vets-api/pull/9180#discussion_r813003500) in this context, isn't `EvssSubmission` also a Sidekiq::Batch callback class? Given how it's being used in [Form526Submission#start](https://github.com/department-of-veterans-affairs/vets-api/pull/9180/files#diff-ada3116cb66cda03a293e8c804592ccb3bc181f3e64689c66fe27d19b0d3cad1R47).

Reply:
Yes, it is. Unlike the other 2 `*JobHandler`, its `start_evss_submission` class method is called directly in various locations, as you are aware.